addons/quickmount/functions/fnc_getInNearest.sqf

```diff
@@ -93,7 +97,7 @@ if (!isNull _target &&
                         {
                             if ((_x select 2) == _cargoIndex) exitWith {_cargoActionIndex = _forEachIndex};
                         } forEach (fullCrew [_target, "cargo", true]);
```

Comment:
This should be a `findIf` ;)

Reply:
:laughing: yeah but out of this PR scope :tongue: